layouts/404.html

```diff
@@ -0,0 +1,8 @@
+{{ define "main"}}
+<main id="main">
+    <div>
+        <h1 id="title"><a href="{{ "/" | relURL }}">Go Home</a></h1>
```

Comment:
Need to support multilang for link, {{ .Site.BaseURL | relLangURL }}

Reply:
Thanks for your feedback. I have updated it.